apps/keyboard/js/imes/latin/predictions.js

```diff
@@ -969,4 +987,4 @@ var Predictions = function() {
     setNearbyKeys: setNearbyKeys,
     predict: predict
   };
-}();
```

Comment:
I changed that from `}();` to `};` such that it's no longer IFFE'ed; we need multiple instances of `Predictions` now.